modelskill/comparison/_collection.py

```diff
@@ -9,7 +9,8 @@
 
 
 from .. import metrics as mtr
-from ..plot import taylor_diagram, TaylorPoint
+from ..plotting import taylor_diagram
+from ..plotting._taylor_diagram import TaylorPoint
```

Comment:
Would it be better to add TaylorPoint to the `__init__` of the plotting module? It somehow seems wrong that the comparison module needs to access a class in a *private* module inside plotting? 🤔